### ISSUE TITLE
What is `JsonNullValueInput` ?

### ISSUE THREAD

**Issue by gaetschwartz** [odroe/prisma-dart#209]
When generating the models, it generated this Create class for my model `Accountconnection`, but instead of `dynamic` (which would represent a json object) that I expected, it takes `JsonNullValueInput` as an argument for the `data` field. 

```prisma
model AccountConnection {
  id             String         @id @db.Uuid
  user_id        String         @db.Uuid
  account_id     String
  provider       String
  data           Json           @db.Json
  email          String?
  users          CalendyUser    @relation(fields: [user_id], references: [user_id], onDelete: NoAction, onUpdate: NoAction)
  user_calendars UserCalendar[]

  @@unique([account_id, user_id, provider], map: "account_connections_unique_constraint")
  @@map("account_connections")
}
```

```dart
enum JsonNullValueInput implements _i1.PrismaEnum {
  @JsonValue('JsonNull')
  jsonNull('JsonNull');

  const JsonNullValueInput([this.originalName]);

  @override
  final String? originalName;
}

/.../

@_i1.jsonSerializable
class AccountConnectionCreateInput implements _i1.JsonSerializable {
  const AccountConnectionCreateInput({
    required this.id,
    required this.accountId,
    required this.provider,
    required this.data,
    this.email,
    required this.users,
    this.userCalendars,
  });

  factory AccountConnectionCreateInput.fromJson(Map<String, dynamic> json) =>
      _$AccountConnectionCreateInputFromJson(json);

  final String id;

  @JsonKey(name: 'account_id')
  final String accountId;

  final String provider;

  final JsonNullValueInput data;

  final String? email;

  final CalendyUserCreateNestedOneWithoutAccountConnectionsInput users;

  @JsonKey(name: 'user_calendars')
  final UserCalendarCreateNestedManyWithoutAccountConnectionsInput?
      userCalendars;

  @override
  Map<String, dynamic> toJson() => _$AccountConnectionCreateInputToJson(this);
}
```

The regular class is generated correctly though.
```dart
@_i1.jsonSerializable
class AccountConnection implements _i1.JsonSerializable {
  const AccountConnection({
    required this.id,
    required this.userId,
    required this.accountId,
    required this.provider,
    required this.data,
    this.email,
  });

  factory AccountConnection.fromJson(Map<String, dynamic> json) =>
      _$AccountConnectionFromJson(json);

  final String id;

  @JsonKey(name: 'user_id')
  final String userId;

  @JsonKey(name: 'account_id')
  final String accountId;

  final String provider;

  final dynamic data;

  final String? email;

  @override
  Map<String, dynamic> toJson() => _$AccountConnectionToJson(this);
}
```

**Comment by medz** [comment]
@gaetschwartz Can you create a minimal example for me? so that I can investigate this issue.

Also: https://github.com/odroe/prisma-dart/issues/212

**Comment by gaetschwartz** [comment]
@medz https://github.com/gaetschwartz/example, here it is :)

**Comment by medz** [comment]
@gaetschwartz Fixed in 3.3.2 version 😊

**Comment by medz** [comment]
Sorry, I haven't been able to fix bugs quickly since last month, but I will try my best to find time to fix bugs and release fixed versions. thanks for waiting
